.eslintrc

```diff
@@ -488,6 +488,31 @@
         ]
       }
     },
+    {
+      "files": [
+        "packages/**/*.js"
+      ],
+      "excludedFiles": [
+        "packages/react/**/*.js"
+      ],
```

Comment:
Also need to exclude WP-specific packages like activation-notice, stories-block, etc.